libr/core/cconfig.c

```diff
@@ -2649,7 +2667,9 @@ R_API int r_core_config_init(RCore *core) {
 	SETCB ("io.buffer", "false", &cb_iobuffer, "Load and use buffer cache if enabled");
 	SETI ("io.buffer.from", 0, "Lower address of buffered cache");
 	SETI ("io.buffer.to", 0, "Higher address of buffered cache");
-	SETCB ("io.cache", "false", &cb_iocache, "Enable cache for io changes");
+	SETCB ("io.cache", "false", &cb_io_cache, "Change both of io.cache.{read,write}");
```

Comment:
was it necessary to change the description? just curios

Reply:
I changed it b/c I did not find a better description. It is edge-triggered, not level-triggered.